signalfx-tracing/signalfx-python-tracing/django/client.py

```diff
@@ -0,0 +1,111 @@
+# IMPORTANT additions to a client file:
+# from signalfx_tracing import auto_instrument, create_tracer, trace
+# from signalfx_tracing.libraries import requests_config
+#
+# requests_config.propagate = True
+# tracer = create_tracer(service_name='<APP_NAME>')
+# auto_instrument(tracer)
+#
+# #Then in your main function, at the end of all function calls to be traced:
+# tracer.close()
+#
+# These are noted below with additional explanations.
+
+
+import logging
+import os
+import requests
+from requests.auth import HTTPBasicAuth
+from signalfx_tracing import auto_instrument, create_tracer, trace
+# Important: Needed to propagate traces
+from signalfx_tracing.libraries import requests_config
+
+
+# Create and configure logger
+if not os.path.exists('log'):
+    os.makedirs('log')
+logging.basicConfig(level='DEBUG',
+                    filename="log/client_log.log",
+                    format='%(asctime)s %(message)s',)
+
+
+# create an instance of the SignalFx tracer
+# and configure auto-instrumentation of the tracing instance
+tracer = create_tracer(service_name='BlogClient')
+auto_instrument(tracer)
+
+
+# Setting requests_config.propagate to 'True'  ensures
+# that the relational information between traces is maintained
+# and forwarded.
+requests_config.propagate = True
```

Comment:
This should come before `auto_instrument`

Reply:
OK. It seemed to work when I tested it, but I didn't delve into the traces sent. Will change it.

Reply:
Changed it